GOVERNANCE.md

```diff
@@ -0,0 +1,131 @@
+
+Consensus decisions can never override or go against the spirit of an earlier explicit vote.
+
+If any [team member](#team-members) raises objections, the team members work together towards a solution that all involved can accept. This solution is again subject to lazy consensus.
+
+In case no consensus can be found, but a decision one way or the other must be made, any [team member](#team-members) may call a formal [majority vote](#majority-vote).
+
+### Majority vote
+Majority votes must be called explicitly as a new topic in the appropriate category in the [Fn Project forums](https://forums.fnproject.io). The topic must be tagged with the 'majority-vote' tag. In the body, the call to vote must state the proposal being voted on. It should reference any discussion leading up to this point.
+
+Votes may take the form of a single proposal, with the option to vote yes or no, or the form of multiple alternatives.
+
+A vote on a single proposal is considered successful if more vote in favor than against.
+
+If there are multiple alternatives, members may vote for one or more alternatives, or vote "no" to object to all alternatives. It is not possible to cast an "abstain" vote. A vote on multiple alternatives is considered decided in favor of one alternative if it has received the most votes in favor, and a vote from more than half of those voting. Should no alternative reach this quorum, another vote on a reduced number of options may be called separately.
+
+### Supermajority vote
+Supermajority votes must be called explicitly as a new topic in the appropriate category in the [Fn Project forums](https://forums.fnproject.io). The topic must be tagged with the 'supermajority-vote' tag. In the body, the call to vote must state the proposal being voted on. It should reference any discussion leading up to this point.
+
+Votes may take the form of a single proposal, with the option to vote yes or no, or the form of multiple alternatives.
+
+A vote on a single proposal is considered successful if at least two thirds of those eligible to vote vote in favor.
+
+If there are multiple alternatives, members may vote for one or more alternatives, or vote "no" to object to all alternatives. A vote on multiple alternatives is considered decided in favor of one alternative if it has received the most votes in favor, and a vote from at least two thirds of those eligible to vote. Should no alternative reach this quorum, another vote on a reduced number of options may be called separately.
+
+## FAQ
+
+This section is informational. In case of disagreement, the rules above overrule any FAQ.
+
+### How do I propose a decision?
+Propose it in the [Fn Project forums](https://forums.fnproject.io/). If there is no objection within a reasonable amount of time, consider the decision made. If there are objections and no consensus can be found, a vote may be called by a team member.
```

Comment:
"reasonable" is pretty subjective

Reply:
Would a concrete timeframe make more sense? 7 days?

Reply:
yes, 7 days seems reasonable and it seems reasonable to spell that out (say, somebody takes a friday off and something objectionable slides in the 36h they didn't check their email! world ends!)